File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/template_rename_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'template_rename'

i = Template_rename::IFoo.new
d = Template_rename::DFoo.new

a = i.blah_test(4)
b = i.spam_test(5)
c = i.groki_test(6)

x = d.blah_test(7)
y = d.spam(8)
z = d.grok_test(9)

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/template_rename_runme.go
package main

import "./template_rename"

func main() {
	i := template_rename.NewIFoo()
	d := template_rename.NewDFoo()

	_ = i.Blah_test(4)
	_ = i.Spam_test(5)
	_ = i.Groki_test(6)

	_ = d.Blah_test(7)
	_ = d.Spam(8)
	_ = d.Grok_test(9)
}
